### DISCUSSION TITLE
New updates coming...!!

### DISCUSSION THREAD

**Discussion by Jarvis-Ank** [Jarvis-Ank/Re-Vanced#11] [Announcements]
New update to the script is now WIP.
 * [x]  Ability to select older Integrations, CLI and patches versions.

**Comment by Jarvis-Ank** [comment]
> New update to the script is now WIP.
>  * [x]  Ability to select older Integrations, CLI and patches versions.

Tested with latest versions, added 3 previous versions. Please test with multiple versions and provide feedback. Based on the responses I'll merge the branch.

**Comment by Bartek20** [comment]
# New functionality has just been released.
From now on, you can select the program's operating mode (currently Automatic or Google Drive)!!!
* Automatic mode will allow you to fully step away from your computer. <br>While you do something else, the program will download the required files and build you your own YouTube ReVanced client.
* Google Drive mode is a mode that you know so far, in which you have to download the youtube.apk file from the APKMirror website yourself, then upload it to your Google Drive and when running the script, you must provide a link from which the program will be able to download it.
* There is also a manual mode (Work In Progress), which will allow you to automatically start downloading the youtube.apk file from the script, but when you run it, you will have to upload this file directly while working with the program (the program will ask you for it).

**Comment by Bartek20** [comment]
# Testers needed
Feel free to test the new functionality that allows you to choose which application from the revanced catalog you want to build
You can find it here: [Bartek20/Re-Vanced](https://github.com/Bartek20/Re-Vanced/tree/EXPERIMANTAL-Multiple-Apps-Support)

**Reply by Jarvis-Ank** [reply]
Hey @Bartek20 there is already a Work In Progress (WIP) branch, under the main branch to Test out code and to make it easy to fix bugs. working on the WIP branch and pushing only stable code to the main branch could reduce unnecessary commits to main branch. The WIP branch is currently behind the main branch, use it for the new features and we can merge them once it's tested.
